src/browsers/stories/heatmap/Heatmap.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
```

Comment:
Minor point, for consistency it would be good to rename the storybook file names to match the component name.